test.js

```diff
@@ -1,27 +1,20 @@
-'use strict';
-var test = require('ava');
-var ghUser = require('./');
+import test from 'ava';
+import fn from './';
 
-test('user exists', function (t) {
-	return ghUser('sindresorhus').then(function (user) {
-		t.is(user.login, 'sindresorhus');
-	});
+test('user exists', async t => {
+	const user = await fn('sindresorhus');
+	
+	t.is(user.login, 'sindresorhus');
 });
 
-test('user doesn\'t exist', function (t) {
-	return ghUser('sindr555esorhus').catch(function (err) {
-		t.is(err.message, 'Response code 404 (Not Found)');
-	});
+test('user doesn\'t exist', async t => {
+	await t.throws(fn('sindresorhus'), Error('Response code 404 (Not Found)'));
```

Comment:
You're right, did this yesterday evening and overlooked this one. Also, the name passed in is not correct (`sindresorhus` -> `sindr555esorhus')

Same issue as below here though, except something extra pops up. Even with a regex, this test doesn't work.

I found some things in AVA why this doesn't work. Will create a new issue there.